pylint/testutils/lint_module_test.py

```diff
@@ -159,18 +159,23 @@ def _open_source_file(self) -> TextIO:
             return open(self._test_file.source, encoding="latin1")
         return open(self._test_file.source, encoding="utf8")
 
-    def _get_expected(self) -> Tuple["MessageCounter", List[OutputLine]]:
+    def _get_expected(
+        self, check_end_position: bool
+    ) -> Tuple["MessageCounter", List[OutputLine]]:
```

Comment:
```suggestion
    ) -> TestMessages:
```
We could maybe create an alias for `` Tuple["MessageCounter", List[OutputLine]]``. Finding a name is hard.

Reply:
`MessageCounter` is already an alias. Personally I think this is just under the limit of it being too long to read. Especially since it is on a new line anyway.

Reply:
Fair enough.